drivers/serial/Kconfig.ns16550

```diff
@@ -72,6 +72,11 @@ config UART_NS16550_SIMULT_ACCESS
 	  When enabled, NS16550 supports IO, MMIO, PCIe UART devices simultaneously.
 	  For io-mapped instances, io-mapped DTS property need to be added in dtsi.
 
+config UART_NS16550_AUTO_INIT_LEVEL
```

Comment:
This name is not what your commit is doing. It is not `AUTO`. You are changing the init level based on whether it is on PCI bus or not. Please rename it accordingly.